src/go/rdctl/pkg/factoryreset/factory_reset_windows.go

```diff
@@ -30,10 +31,48 @@ import (
 	"golang.org/x/text/encoding/unicode"
 )
 
-func KillRancherDesktop() {
+func KillRancherDesktop() error {
 	cmd := exec.Command("taskkill", "/IM", "Rancher Desktop.exe", "/T", "/F")
 	cmd.SysProcAttr = &syscall.SysProcAttr{CreationFlags: CREATE_NO_WINDOW}
-	cmd.Run()
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("Failed to taskkill Rancher Desktop.exe: %w", err)
+	}
+	return nil
+}
+
+//WARNING: This will fail if we localize the name of the app
+
+func CheckProcessWindows() (bool, error) {
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+
+	cmd := exec.Command("tasklist", "/FI", "WINDOWTITLE eq Rancher Desktop", "/FO", "List")
+	cmd.SysProcAttr = &syscall.SysProcAttr{CreationFlags: CREATE_NO_WINDOW}
+	allOutput, err := cmd.CombinedOutput()
+	if err != nil {
+		return false, fmt.Errorf("Failed to run %q: %w", cmd, err)
+	}
+	if strings.HasPrefix(string(allOutput), "INFO: No tasks are running which match the specified criteria") {
+		logrus.Warnf("%s\n        %s\n%s",
+			"Looking to see if Rancher Desktop is running, got info message:",
+			string(allOutput),
+			"Assuming Rancher Desktop is no longer running.")
+		return false, nil
+	}
+	r := csv.NewReader(strings.NewReader(string(allOutput)))
```

Comment:
I'm confused; we're using `/FO List` but treating it as CSV?

Reply:
I was confused. My head wanted `"CSV"` but my hands typed `"LIST"`